source/essences.textile

```diff
@@ -9,6 +9,30 @@ Essences are normal Rails models, so it is pretty easy to "add your own":create_
 
 endprologue.
 
+h3. Hints, Defaults, As Element Title
+
+When defining elements, you can assign hints and defaults to every essence like this:
```

Comment:
I think only talking of "defaults" is not clear enough. Maybe talking about "default values" is better?